docker/root/usr/local/bin/config-cli

```diff
@@ -21,7 +21,18 @@ if ! command -v curl >/dev/null; then
   exit 1
 fi
 
+if [ -z "${KEYCLOAK_SSLVERIFY+x}" ]; then
+    VERIFY=true
+else
+    VERIFY="$(echo ${KEYCLOAK_SSLVERIFY}|tr '[A-Z]' '[a-z]')"
+fi
+
 echo "Wait until keycloak is available ..."
-wtfc -T "${WAIT_TIME_IN_SECONDS:-120}" -S 0 -I 2 curl -f "${KEYCLOAK_URL}"
+if [ "${VERIFY}" = "false" ]; then
+    echo "TLS will be ignored..."
+    wtfc -T "${WAIT_TIME_IN_SECONDS:-120}" -S 0 -I 2 curl -f "${KEYCLOAK_URL} -k"
```

Comment:
Quoting looks wrong. Does it work?

```suggestion
    wtfc -T "${WAIT_TIME_IN_SECONDS:-120}" -S 0 -I 2 curl -f "${KEYCLOAK_URL}" -k
```

Reply:
indeed that's wrong. it worked anyway. i changed this.